itests/standalone/basic/src/test/java/org/wildfly/camel/test/milo/MiloServerIntegrationTest.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.camel.test.milo;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.impl.DefaultCamelContext;
+import org.eclipse.milo.opcua.stack.core.types.builtin.DataValue;
+import org.eclipse.milo.opcua.stack.core.types.builtin.Variant;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extension.camel.CamelAware;
+
+
+/**
+ * Unit tests for milo server component without using an actual connection
+ */
+@CamelAware
+@RunWith(Arquillian.class)
+public class MiloServerIntegrationTest {
+
+    @Deployment
+    public static JavaArchive createDeployment() {
+        JavaArchive archive = ShrinkWrap.create(JavaArchive.class, "milo-server-tests.jar");
+        return archive;
+    }
+
+    @Test
+    public void testAcceptData() throws Exception {
+        
+        CamelContext camelctx = new DefaultCamelContext();
+        camelctx.addRoutes(new RouteBuilder() {
+            public void configure() throws Exception {
+                from("milo-server:myitem1")
+                .to("mock:test");
+            }
+        });
+
+        camelctx.start();
+        try {
+            MockEndpoint mockTest = camelctx.getEndpoint("mock:test", MockEndpoint.class);
+
+            ProducerTemplate producer = camelctx.createProducerTemplate();
+
+            sendBody(producer, new Variant("Foo"));
+            Assert.assertTrue("Expect no failures", mockTest.getFailures().isEmpty());
```

Comment:
Is such an assertion meaningful? Should we not be validating that whatever message body was received from the milo-server endpoint, eventually made it to the mock:test endpoint.